bin/update-license-files.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+curl https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json > ../licenses.json
```

Comment:
Maybe add a note to the readme about running this, and add the exec flag to the file (don't know how to check that on github)